### PR TITLE
Update de.toml

### PR DESCRIPTION
**Pull request by BrunoFischerGermany**
I have added new abbreviations.
rd - rund
hl - heilig
dgl - der gleichen
No. - Number
Abk - Abkürzung
Abb. - Abbildung
geb. - geboren
gegr. - gegründet

### CODE REVIEW

**Comment by stefangrotz** [comment]
Danke für den Pull Request! 

dgl würde ich zu "dergleichen" statt "Der gleichen" machen.

@MichaelKohler  fällt Dir noch etwas auf?

**Comment by MichaelKohler** [comment]
Apart from Stefan's comment, this looks good to me. I'm also gonna have a look at the sample output to make sure this has no unintended side effects once that change is made. 

Thanks!

**Comment by BrunoFischerGermany** [comment]
Ich habe noch 2 Abkürzungen beim Spenden meiner Stimme gefunden.

vs - versus und kgl königlich

Viele Grüße

File: src/rules/de.toml [hunk 1]
```diff
@@ -32,6 +32,8 @@ replacements = [
   ["Abb.", "Abbildung"],
   ["geb.", "geboren"], 
   ["gegr.", "gegründet"],
+  ["vs.", "versus"],
+  ["kgl", "königlich"],
```
Comment (MichaelKohler) [inline src/rules/de.toml:36]: Is this really occuring without a period? I wonder how many words there are that have "kgl" in the middle somewhere, which would be replaced as well 😅 nothing comes to mind right now, but might not be safe

Comment (BrunoFischerGermany) [reply src/rules/de.toml:36]: Oh maybe is it better with kgl. 

I found 

eidg. - Eidgenossenschaftlich

Maybe add this Zoo?

File: src/rules/de.toml [hunk 2]
```diff
@@ -24,6 +24,16 @@ replacements = [
   ["evtl.", "eventuell"],
   ["bzgl.", "bezüglich"],
   ["Nr.", "Nummer"],
+  ["rd.", "rund"], 
+  ["hl.", "heilig"],
```
Comment (MichaelKohler) [inline src/rules/de.toml:28]: Just came to my mind, I don't think this one will work nicely. I generally think that adjectives won't work nicely due to the different cases.

"Die hl. drei Könige" -> heiligen
"Die hl. Maria" -> heilige

File: src/rules/de.toml [hunk 3]
```diff
@@ -24,6 +24,16 @@ replacements = [
...
+  ["dgl.", "Der gleichen"],
```
Comment (MichaelKohler) [inline src/rules/de.toml:29]:
```suggestion
  ["dgl.", "dergleichen"],
```

File: src/rules/de.toml (hunk 1 above)
Comment (MichaelKohler) [reply src/rules/de.toml:36]: See my comment above, same applies here. And I think the same would also apply for "eidg.". What do you think?

**Review by MichaelKohler — CHANGES_REQUESTED** [review]
Sorry for the delay here. Added a few comments.

File: src/rules/de.toml (hunk 1 above)
Comment (BrunoFischerGermany) [reply src/rules/de.toml:36]: > Is this really occuring without a period? I wonder how many words there are that have "kgl" in the middle somewhere, which would be replaced as well 😅 nothing comes to mind right now, but might not be safe

you are right. Not without A Period.

So "kgl." And "Eidg." Are better. 👌

**Pull request closed without merging**